assets/css/v2/lcd_common/departures/later_departures.scss

```diff
@@ -0,0 +1,83 @@
+@use "sass:math";
+
+.later-departures {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 280px;
+  background-color: $warm-neutral-90;
+
+  &__header {
+    box-sizing: inherit;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 32px;
+    border-bottom: 4px solid $true-grey-70;
+
+    h3 {
+      box-sizing: inherit;
+      flex-shrink: 0;
+      padding: 18px 0;
+      margin: 0;
+      font-family: Inter;
+      font-size: 40px;
+      font-weight: 500;
+      line-height: 43px;
+      color: $cool-black-15;
+      text-transform: uppercase;
+      letter-spacing: 2px;
+    }
+
+    ol {
+      display: flex;
+      flex-direction: row;
+      gap: 24px;
+      justify-content: space-between;
+      list-style: none;
+
+      .route-pill {
+        position: static;
+        box-sizing: border-box;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: auto;
+        min-width: 86px;
+        height: 44px;
+      }
+
+      .route-pill--yellow {
+        box-shadow: 1px 1px 1px 0 rgba(0 0 0 / 25%);
+      }
+
+      .route-pill__text {
+        position: static;
+        padding: 0 14px;
+        font-size: 36px;
+        line-height: 36px;
+      }
+    }
+  }
+
+  &--departure--selected {
+    position: relative;
+
+    &::after {
+      $carot-size: 16px;
+
+      position: absolute;
+      bottom: -25px;
+      left: calc(50% - math.hypot($carot-size, $carot-size) / 2);
```

Comment:
I have no doubt I'm going to be impressed by this, but I will first need an explanation of how and why the math works 😄

Reply:
If I remove the `left` and `transform` values we have a square that is aligned with the left side of the bounding box around the route pill

<img width="228" alt="Screenshot 2024-06-03 at 10 59 04 AM" src="https://github.com/mbta/screens/assets/1699281/716da5d5-e83d-47ef-836b-41af3eba368a">

Then if I set `left: 50%` the square isn't centered within the bounding box of the route pill but instead its left side is.

<img width="191" alt="Screenshot 2024-06-03 at 10 59 18 AM" src="https://github.com/mbta/screens/assets/1699281/d1c6904b-d9b7-4256-8bb8-03ffc82351ea">

So I need to set a left value slightly less than `50%`: `50% - half of the width of my caret`. This would simply be `calc(50% - $caret-size / 2)` _if_ I wasn't also going to rotate it 45deg...

<img width="202" alt="Screenshot 2024-06-03 at 10 59 23 AM" src="https://github.com/mbta/screens/assets/1699281/eccd8a2a-dabe-4ffa-b1cf-21930f3c2393">

By rotating it its width is now something other than `$caret-size` (`16px`)

<img width="460" alt="Screenshot 2024-06-03 at 11 04 52 AM" src="https://github.com/mbta/screens/assets/1699281/60bd3cb5-c2f0-4ac3-a9d9-0f4cffb16238">

I _could_ just copy this computed value into the CSS but that makes it difficult to ever change the size of the caret. Luckily this value is actually the hypotenuse of one of the two triangles that make up this square! $\sqrt(16px) + \sqrt(16px) = \sqrt(box width)$. Good ol' Pythagoras helping us out!

Finally, SCSS's `math.hypot` will solve this equation for us. Now if we change the size of the caret it will remain horizontally centered beneath the route pill.

---

An aside: Is this the _best_ way to do this? I'm not sure! Is it the way I thought up first? Yes, it is. If this is too strange I'm happy to noodle on other approaches **or** if you have an idea I'm happy to try that out too!!

Reply:
You know it's a good explanation when you have to break out the LaTeX 😄 Thanks for going over this, and it's good to also have it documented here with visuals.